plugins/outputs/groundwork/groundwork_test.go

```diff
@@ -0,0 +1,94 @@
+package groundwork
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/gwos/tcg/sdk/clients"
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/testutil"
+	"github.com/stretchr/testify/require"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"testing"
```

Comment:
Other plugins usually have this kind of "groups": builtin, external, internal:
```suggestion
	"encoding/json"
	"fmt"
	"io/ioutil"
	"net/http"
	"net/http/httptest"
	"testing"

	"github.com/gwos/tcg/sdk/clients"
	"github.com/stretchr/testify/require"

	"github.com/influxdata/telegraf"
	"github.com/influxdata/telegraf/testutil"
```

Reply:
Done